cms/djangoapps/contentstore/utils.py

```diff
@@ -160,7 +160,10 @@ def get_lms_link_for_certificate_web_view(user_id, course_key, mode):
 
 def course_image_url(course):
     """Returns the image url for the course."""
-    loc = StaticContent.compute_location(course.location.course_key, course.course_image)
+    try:
+        loc = StaticContent.compute_location(course.location.course_key, course.course_image)
+    except InvalidKeyError:
+        return ''
     path = StaticContent.serialize_asset_key_with_slash(loc)
     return path
```

Comment:
As long as that gives the correct behavior on the frontend, that's fine (although from a local perspective, it might be clearer to either just let the exception escape, or return None)